src/Severity.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+/*
+ * This file is part of Raven.
+ *
+ * (c) Sentry Team
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sentry;
+
+/**
+ * This class represents an enum of severity levels an event can be associated
+ * to.
+ *
+ * @author Stefano Arlandini <sarlandini@alice.it>
+ */
+final class Severity
+{
+    /**
+     * This constant represents the "debug" severity level.
+     */
+    public const SEVERITY_DEBUG = 'debug';
```

Comment:
Isn't the `SEVERITY_` prefix a little redundant? It's `Severity:SEVERITY_DEBUG` like this...